### PR TITLE
Dasepho metal strip

### PR DESCRIPTION
**Pull request by Dasepho**
Please review, particularly for any game-specific details I might need to add

### CODE REVIEW

**Comment by trickeydan** [comment]
Have we explained the purpose of pull-up / down resistors elsewhere?

**Comment by Dasepho** [comment]
Screenshot at 2019-07-30 15-24-54
This is the relevant section of the labs, but it's a bit lacking. I'll elaborate on it

File: electronics/sections/metal_strip.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - Reliability
+as well as other factors based on your specific project. Which method do you think is the best for your robot?
+
+- Try to improve your counting circuit / code to deal with switch bounce. If you use a capacitor, this is where it should go:
+
+\begin{center} \includegraphics[height=8cm]{img/debounced_metal_strip.png} \end{center}
+*Sensor circuit with capacitor for debouncing*
+
+
+## Simon Says (Extension)
```
Comment (trickeydan) [inline electronics/sections/metal_strip.md]: Can we remove this section for now?

They can't click on the link, and it doesn't work with either the SR kit or the kit we plan to ship.

An issue should be opened to track and re-add this if we have time

Comment (Dasepho) [reply electronics/sections/metal_strip.md]: Sure. I didn't really know about this section anyway and was just assuming it was something to be left in. I'll take it out and make an issue to look at it later

**Pull request merged**